src/main/java/io/vlingo/symbio/store/dispatch/Dispatchable.java

```diff
@@ -21,9 +33,21 @@
    */
   private final LocalDateTime createdOn;
 
-  protected Dispatchable(String id, LocalDateTime createdOn) {
+  /**
+   * My R concrete {@code State<?>} type.
+   */
+  private final RS state;
+
+  /**
+   * My {@code List<Entry<?>>} to dispatch
+   */
+  private final List<E> entries;
+
+  public Dispatchable(String id, LocalDateTime createdOn, RS state, List<E> entries) {
```

Comment:
Suggestion: It's best to declare (almost) all method parameters as `final`, assuming that you never change the reference internally.